mmv1/templates/terraform/examples/external_vpn_gateway.tf.erb

```diff
@@ -99,3 +100,4 @@ resource "google_compute_router_peer" "router1_peer2" {
   advertised_route_priority = 100
   interface                 = google_compute_router_interface.router1_interface2.name
 }
+# [START cloudvpn_ha_external]
```

Comment:
What are these START and START identifiers ?

Should this be START and START enclosing this?

Reply:
Good catch It should be START and END. These are region tags, which tell DevSite which part of the code to include on the website. In this case, we want to include all of it, so I have enclosed all of it with these tags. My next commit will fix the closing tag.

Reply:
By the way, the expert on this workflow is camthornton@ (on github, he is c2thorn).